ChaosMod/Memory/Hooks/HandleToEntityStructHook.cpp

```diff
@@ -10,13 +9,15 @@ __int64 _HK_HandleToEntityStruct(Entity entity)
 	{
 		return 0;
 	}
-
-	if (entity == m_origHandle)
+	Entity vehToContinue = entity;
+	while (vehicleMap.count(vehToContinue) > 0)
 	{
-		return _OG_HandleToEntityStruct(m_proxyHandle);
+		vehToContinue = vehicleMap[vehToContinue];
+    	if (vehToContinue <= 0) {
```

Comment:
The bracket here should be on a new line

Reply:
That's fixed but now its indentation is broken 😅

Reply:
oh the joy of editing via mobile 👌